src/main/config/parameter_group_ids.h

```diff
@@ -66,6 +66,8 @@
 #define PG_MSP_SERVER_CONFIG 48
 #define PG_VOLTAGE_METER_CONFIG 49
 #define PG_AMPERAGE_METER_CONFIG 50
+#define PG_MOTOR_CONFIG 51
```

Comment:
Did you consider renaming PG_MOTOR_AND_SERVO_CONFIG to PG_MOTOR_CONFIG  and bumping record version?
I'm not sure which one is better ...

Adding dummy value (to replace servoCenterPulse) into motor config will even allow automatic upgrade ...